### ISSUE TITLE
GRASS temporal programs not working on OS X Brew Install

### ISSUE THREAD

**Issue by mankoff** [OSGeo/homebrew-osgeo4mac#133]
I have `grass 7.0.3` installed on OS X via homebrew thanks to #118. Everything appears to work fine for several weeks, but I never tested new (in v. 7.0) temporal commands. They do not work. For example:

```
t.create --help
```

causes:

```
Traceback (most recent call last):
  File "/usr/local/Cellar/grass-70/7.0.3/grass-7.0.3/scripts/t.create", line 62, in <module>
    import grass.temporal as tgis
  File "/usr/local/Cellar/grass-70/7.0.3/grass-7.0.3/etc/python/grass/temporal/__init__.py", line 1, in <module>
    from core import *
  File "/usr/local/Cellar/grass-70/7.0.3/grass-7.0.3/etc/python/grass/temporal/core.py", line 38, in <module>
    from c_libraries_interface import *
  File "/usr/local/Cellar/grass-70/7.0.3/grass-7.0.3/etc/python/grass/temporal/c_libraries_interface.py", line 19, in <module>
    import grass.lib.gis as libgis
  File "/usr/local/Cellar/grass-70/7.0.3/grass-7.0.3/etc/python/grass/lib/gis.py", line 23, in <module>
    _libs["grass_gis.7.0.3"] = load_library("grass_gis.7.0.3")
  File "/usr/local/Cellar/grass-70/7.0.3/grass-7.0.3/etc/python/grass/lib/ctypes_loader.py", line 57, in load_library
    raise ImportError,"%s not found." % libname
ImportError: grass_gis.7.0.3 not found.

```

**Comment by dakcarto** [comment]
Are you using Homebrew or system Python?

**Comment by mankoff** [comment]
I'm using Homebrew Python. `brew doctor`'s only complaint is that I have a case sensitive filesystem.

**Comment by rkrug** [comment]
I can confirm this error. Also using homebrew python (I think...).

**Comment by mankoff** [comment]
I've just solved this (using Fink-installed GRASS 7.2.0) with:

    export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$GISBASE/lib"

And then `t.create` works.
